### PR TITLE
initial edited upload.

### PR DESCRIPTION
**Pull request by zflategraff**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang=“en”>
```
Comment (mbMosman) [inline index.html]: The "smart quotes" are causing validation errors for this page.  You might want to switch to an editor that does not use those.

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+<head>
+<title>CCIS 1301 Fan Page - ANI DIFRANCO - Zachary Flategraff</title>
+<meta charset=“utf-8”>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+<h1>Ani Difranco-</h1>
```
Comment (mbMosman) [inline index.html]: You should use the `<header>` element with your `<h1>`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <img src="http://cdn.shopify.com/s/files/1/0153/0645/products/06_10_14_st_louis_mo_061_s_alzner_1024x1024.jpg?v=1453398449" width="30%" alt="Ani Difranco Live Picture">
+<pre>(photo courtesy anidifranco.com)</pre>
```
Comment (mbMosman) [inline index.html]: Nicely done with the attribution!

**Comment by mbMosman** [comment]
Nice work, be careful with the validation though.  I see you got no errors for your screenshot.  You might want to validate using the URI once you have pushed to GitHub.  Make sure that you are able to see the same issues.  

Writing this it occurs to me that maybe that changed when you did part 2.  I'll check the original commit to see if the same issues persist, but you should definitely fix this and resubmit (new pull-request) for part 2.

**Comment by mbMosman** [comment]
FYI - It looks like the smart quotes are present in all of the commits...

**Comment by zflategraff** [comment]
Hey Mary- I am not sure where the smart quotes are coming from. Im using a mac, brackets and terminal to do all my git work.. Do do you have an idea of whefe i should start looking?

Thank you!
Zac

**Comment by mbMosman** [comment]
I can't say how you are getting them, but your quotes from the <html lang="en"> and the <meta charset="utf-8"> are the invalid, smart quotes.  However the rest of your document is all fine normal quotes.  If you had copy/pasted that in from somewhere, maybe that is where they came from?  I can't say. 

I'll let these go for this week, but make sure to get them cleaned up.  View your source on the web browser to see if they are slanted or straight.  
image

**Comment by mbMosman** [comment]
You should also validate by URL so that you can see any errors.  You also want to make sure you have an open pull request each week.  This one was closed because I assumed you would make fixes.

image

**Pull request closed without merging**